src/silx/gui/utils/matplotlib.py

```diff
@@ -41,6 +41,36 @@
 
 from .. import qt
 
+# This must be performed before any import from matplotlib
+if qt.BINDING in ("PySide6", "PyQt6", "PyQt5"):
+    matplotlib.use("Qt5Agg", force=False)
+    from matplotlib.backends.backend_qt5agg import FigureCanvasQTAgg  # noqa
+
+else:
+    raise ImportError("Unsupported Qt binding: %s" % qt.BINDING)
```

Comment:
Moved this first to avoid lazy loading in function

Reply:
I don't understand why you put such comment in the PR.

If it's useful to understand the code, better to put such comment in the code itself, no?

Reply:
It's to explain the modification: It's no longer needed.
A comment in the code saying I first import modules is pointless.